src/librustc_mir/interpret/eval_context.rs

```diff
@@ -768,6 +770,8 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         // FIXME(oli-obk): make this check an assertion that it's not a static here
         // FIXME(RalfJ, oli-obk): document that `Place::Static` can never be anything but a static
         // and `ConstValue::Unevaluated` can never be a static
+        // FIXME(oli-obk, spastorino): the above FIXME is not true anymore, PlaceBase::Static does
+        // not exist anymore (except for promoteds but it's going away soon).
```

Comment:
Then maybe remove the comment instead of leaving a wrong comment around?^^

Reply:
It's not wrong yet

Reply:
You added a comment saying it's "not true any more". "not true" = "wrong". So what now?^^